releasenotes/releasenotes_develop.md

```diff
@@ -1,52 +1,343 @@
+- DockerHub: Stale images are going to be deleted soon [2710](https://github.com/keptn/keptn/issues/2710)
+- Move tests for delivery assistant and self-healing to K3s [2771](https://github.com/keptn/keptn/issues/2771)
+- Reduce the number of platform/integration tests on Travis-CI [2718](https://github.com/keptn/keptn/issues/2718)
+- Makefile: *Fixed* - Build-CLI works, but the resulting binary is not [2504](https://github.com/keptn/keptn/issues/2504)
+- Makefile: Add a way to build all Dockerfile [2464](https://github.com/keptn/keptn/issues/2464)
+- Makefile: Add build and run targets [2405](https://github.com/keptn/keptn/issues/2405)
+
+</p>
+</details>
+
+<details><summary>Fixed CI issues</summary>
+<p>
+
+- *Fixed*: Integration tests are failing (Minishift, self-healing) [3325](https://github.com/keptn/keptn/issues/3325)
+- *Fixed*: Integration tests: GKE clusters are not deleted afterwards in some cases [3243](https://github.com/keptn/keptn/issues/3243)
+- *Fixed*: Flaky integration tests: Integration tests fail (in unpredictable situations) [2149](https://github.com/keptn/keptn/issues/2149)
+- *Fixed*: Integration test stalls at the Keptn auth command [2704](https://github.com/keptn/keptn/issues/2704)
+- *Fixed*: Integration test: Setup of Keptn fails due to server version check [2701](https://github.com/keptn/keptn/issues/2701)
+- *Fixed*: Unable to do remote debugging of mongodb-datastore due to liveness-probe [2536](https://github.com/keptn/keptn/issues/2536)
+- *Fixed*: GitHub Action Reviewdog Fails: The `add-path` command is disabled [2694](https://github.com/keptn/keptn/issues/2694)
+
+</p>
+</details>
 
 ## Good to know / Known Limitations
 
+This section lists bugs and limitations that are known but not fixed in this release. They will get addressed in one of the next releases.
+
+- Keptn CLI can not be used for automation due to Kube context check [3208](https://github.com/keptn/keptn/issues/3208)
+  - There is a workaround provided, please see [here](https://github.com/keptn/keptn/issues/3208#issuecomment-781982765)
+- Creating a project fails on OpenShift due to missing write permissions [2453](https://github.com/keptn/keptn/issues/2453)
+- Hovering over the score in an `approval.triggered` events in the Bridge leads to a scroll-up / jump-up in Firefox [#2369](https://github.com/keptn/keptn/issues/2369)
+- Remove the functionality to listen to service.create.finished event from helm-service [2989](https://github.com/keptn/keptn/issues/2989)
+  - The helm-service does not support listening on `sh.keptn.event.service.delete.finished` events when running on the execution plane. This leads in the limitation that deleting deployed services on the execution plan becomes a manual task. To delete a deployed service, execute: 
+    ```
+    helm ls -n <NAMESPACE>
+    helm delete <HELM_RELEASE> -n <NAMESPACE>
+    ```
```

Comment:
As described in https://github.com/keptn/keptn/issues/2989#issuecomment-782066768, it's not needed to delete helm releases, just the namespace they live in. That has always been a manual step anyway.

Reply:
But when I have multiple services in the same namespace (carts, carts-db) and I deleted just carts-db - then I have to manually delete the Helm Chart for `carts-db`. Otherwise, I delete both.

Reply:
Right, in that case you might want to manually delete just one service. Let's leave it there.